diff --git a/libvita2d/Makefile b/libvita2d/Makefile
index 54db1c3..bd82935 100644
--- a/libvita2d/Makefile
+++ b/libvita2d/Makefile
@@ -12,8 +12,6 @@ SHADERS    = shader/compiled/clear_v_gxp.o shader/compiled/clear_f_gxp.o \
 PREFIX  ?= ${VITASDK}/arm-vita-eabi
 CC      = arm-vita-eabi-gcc
 AR      = arm-vita-eabi-ar
-CFLAGS  = -Wl,-q -Wall -O3 -I$(INCLUDES) -I$(VITASDK)/arm-vita-eabi/include/freetype2 -ffat-lto-objects -flto
-ASFLAGS = $(CFLAGS)
 
 all: $(TARGET_LIB)
 
-- 
2.9.2

